src/modules/webhooks/send-notification/1.0/module.json

```diff
@@ -0,0 +1,228 @@
+{
+    "api": {
+      "url": "https://dev-api.edifydev.net/v4/notifications",
+      "method": "post",
+      "header": {
+        "u": "{{edify.user.email}}:{{edify.user.apiToken}}",
+        "content-type": "application/json"
+      },
+      "onResponse": {
+        "valueAccessor": "",
+        "valueMap": {
+          "any": {
+            "label": "",
+            "actions": [],
+            "exitPort": "",
+            "exitPortValueMap": {},
+            "exitPortValueAccessor": "",
+            "httpCode": "",
+            "collapsed": false
+          }
+        }
+      }
+    },
+    "name": "Send Notification",
+    "sections": [
+      {
+        "label": "Notification",
+        "inputs": [
+          {
+            "name": "type",
+            "type": "dropdown",
+            "label": "Type",
+            "values": [
+              {
+                "label": "HTML5",
+                "value": "html5"
+              },
+              {
+                "label": "Push Notification",
+                "value": "push"
+              },
+               {
+                "label": "Toast Notification",
+                "value": "toast"
+              }
+  
+            ],
+            "defaultValue": "html5"
+          },
+          { 
+            "api": {
+            "url": "https://dev-api.edifydev.net/v4/users",
+            "method": "get",
+            "headers": {
+                "u": "{{edify.user.email}}:{{edify.user.apiToken}}"
+            },
+            "labelFormat": "{{$.lastName}}, {{$.firstName}}",
+            "valueFormat": "{{$.userId}}",
+            "valueSource": "$.users"
+          },
+          "name": "users",
+          "type": "dropdown",
+          "label": "Users",
+          "valuesType": "static",
+          "defaultValue": ""
+          },
+      {
+        "name": "title",
+        "type": "input",
+        "label": "Title",
+        "subtype": "text",
+        "valuesType": "static",
+        "defaultValue": ""
+    },
+    
+    {
+      "name": "body",
+      "type": "input",
+      "label": "Body",
+      "subtype": "text",
+      "valuesType": "static",
+      "defaultValue": ""
+  }
+
+  
+        ]
+      },  
+      {
+        "name": "data",
+        "type": "input",
+        "label": "Data",
```

Comment:
i dont think we need this data section. just put the same condional render on channelId and notificationId